OpenRA.Mods.Cnc/Graphics/ClassicSpriteSequence.cs

```diff
@@ -33,15 +33,13 @@ public class ClassicSpriteSequence : DefaultSpriteSequence
 		static readonly SpriteSequenceField<bool> UseClassicFacings = new SpriteSequenceField<bool>(nameof(UseClassicFacings), false);
 		readonly bool useClassicFacings;
 
-		public ClassicSpriteSequence(ModData modData, string tileSet, SpriteCache cache, ISpriteSequenceLoader loader, string sequence, string animation, MiniYaml info)
-			: base(modData, tileSet, cache, loader, sequence, animation, info)
+		public ClassicSpriteSequence(ModData modData, string tileSet, SpriteCache cache, ISpriteSequenceLoader loader, string image, string sequence, MiniYaml data, MiniYaml defaults)
+			: base(modData, tileSet, cache, loader, image, sequence, data, defaults)
 		{
-			var d = info.ToDictionary();
-			useClassicFacings = LoadField(d, UseClassicFacings);
+			useClassicFacings = LoadField(UseClassicFacings, data, defaults);
 
 			if (useClassicFacings && facings != 32)
-				throw new InvalidOperationException(
-					$"{info.Nodes[0].Location}: Sequence {sequence}.{animation}: UseClassicFacings is only valid for 32 facings");
```

Comment:
Why are you removing the line number?

Reply:
`info.Nodes[0].Location` was often the wrong line! (and the current plumbing doesn't let us easily report the correct one)

This PR removes it for correctness and consistency with the other sequence yaml errors.
#20597 implements line numbers properly for all errors.